scripts/upgrade

```diff
@@ -163,41 +181,11 @@ ynh_system_user_create --username=$app
 #=================================================
 # MODIFY A CONFIG FILE
 #=================================================
+ynh_script_progression --message="Setting up configuration files..." --weight=2
 
-# Check if network.xml exists (introduced in v10.7)
-if [ -f "/etc/jellyfin/network.xml" ]; then
-	ynh_script_progression --message="Configuring the settings..." --weight=1
-
-	ynh_backup_if_checksum_is_different --file="$config_path/network.xml"
-	ynh_backup_if_checksum_is_different --file="$config_path/system.xml"
-
-	# Set permissions to app files
-	chown -R $app: $final_path
-
-	# Load services once to generate network.xml
-	ynh_systemd_action --service_name=$app --action="restart" --log_path="systemd" --line_match="Started Jellyfin Media Server" --timeout=15
-	ynh_systemd_action --service_name=$app --action="stop" --log_path="systemd" --line_match="Stopped Jellyfin Media Server" --timeout=15
-
-	# Port config
-	ynh_replace_string --match_string="<PublicPort>8096</PublicPort>" --replace_string="<PublicPort>$port</PublicPort>" --target_file="$config_path/network.xml"
-	ynh_replace_string --match_string="<HttpServerPortNumber>8096</HttpServerPortNumber>" --replace_string="<HttpServerPortNumber>$port</HttpServerPortNumber>" --target_file="$config_path/network.xml"
-
-	# Enable IPv6
-	ynh_replace_string --match_string="<EnableIPV6>false</EnableIPV6>" --replace_string="<EnableIPV6>true</EnableIPV6>" --target_file="$config_path/network.xml"
-
-	# BaseUrl config
-	ynh_replace_string --match_string="<BaseUrl />" --replace_string="<BaseUrl>$path_url</BaseUrl>" --target_file="$config_path/network.xml"
-
-	ynh_store_file_checksum --file="$config_path/network.xml"
-	ynh_store_file_checksum --file="$config_path/system.xml"
-fi
-
-# Lower logging verbosity
-# But first, if missing, copy it from its template
-if [ ! -f "$config_path/logging.json" ]; then
-	cp "$config_path/logging.default.json" "$config_path/logging.json"
-fi
-ynh_exec_fully_quiet ynh_replace_string --match_string="\"Default\": \"Information\"" --replace_string="\"Default\": \"Error\"" --target_file="$config_path/logging.json"
+ynh_add_config --template="system.xml" --destination="$config_path/system.xml"
+ynh_add_config --template="network.xml" --destination="$config_path/network.xml"
+ynh_add_config --template="logging.json" --destination="$config_path/logging.json"
```

Comment:
It's kinda dangerous as it overwrites the modifications the administrator did… but I guess it's better than before :)

Reply:
Yeah I know... I think it will be become very annoying for the user. I may revert that.

Reply:
I'm randomly thinking about this and realize we could have a question during upgrade asking wether or not to overwrite manually-modified files (so instead, the diff would be displayed + the new conf kept somewhere, ie the reverse behavior which is currently to overwrite but keep the old conf elsewhere)

Reply:
I guess for now it's better than currently on master though :)

Reply:
#88 includes a commit that prevents overriding system.xml. It should be fine for the time being. :)